src/test/groovy/func/GradleCompatibilitySpec.groovy

```diff
@@ -33,7 +33,7 @@ class GradleCompatibilitySpec extends GatlingFuncSpec {
         then:
         result.task(":tasks").outcome == SUCCESS
         where:
-        gradleVersion << ["4.0.1", "4.10.2", "5.0", "5.6.4", "6.0", "6.3"]
+        gradleVersion << ["4.0.1", "4.10.2", "5.0", "5.6.4", "6.0", "6.3", "6.4.1", "6.6.1"]
```

Comment:
There's no need to add `6.6.1` here. 

Because the functional tests will use it implicitely.
So GradleCompatibilitySpec should test not the current version, but the last meaningful previous version.

Reply:
I suspected as much, but wasn't sure. Will remove thanks!